middlewares/injectRenderAsync.js

```diff
@@ -0,0 +1,28 @@
+const Pool = require('threads').Pool;
+
+const pool = new Pool(10);
```

Comment:
I just picked a number that seemed reasonable - this should probably be configurable though?

Reply:
Default without specifying is number of CPUs.